project/AkkaBuild.scala

```diff
@@ -1480,9 +1480,10 @@ object AkkaBuild extends Build {
     )
     def defaultImports = Seq("!sun.misc", akkaImport(), configImport(), scalaImport(), "*")
     def akkaImport(packageName: String = "akka.*") = versionedImport(packageName, "2.3", "2.4")
-    def streamAndHttpImport(packageName: String) = versionedImport(packageName, "1.0", "1.1")
+    def streamAndHttpImport(packageName: String) = versionedImport(packageName, "2.0", "2.1")
```

Comment:
@ktoso I couldn't find this change on master. It definitely needs to be there, will look again after some sleep, want to get it in at the same time. Thanks!